webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -4899,6 +4899,14 @@ var MCK_BOT_MESSAGE_QUEUE = [];
                         mckMessageService.getAndSetAwayMessage({}, params, conversationAssigneeDetails, isAgentOffline);
                         mckMessageService.updateConversationHeader(updateConversationHeaderParams);
                     });
+                    for (var key in params.groupDetails.users) {
```

Comment:
@hardikparashar is it mandatory that params. groupDetails will always have some value?